[android][expo-image-picker] content:// URIs are not usable in `ExponentImagePicker.launchImageLibraryAsync`

### Minimal reproducible example

I don't have Android. I have no idea what is an `uri` with `content://`

### Summary

"expo-image-picker": "~14.3.2",

Exactly as it's happening here https://github.com/expo/expo/issues/9354, I'm getting an error in `ExponentImagePicker.launchImageLibraryAsync`

```
Call to function 'ExponentImagePicker.launchImageLibraryAsync' has been rejected.
→ Caused by: java.lang.IllegalArgumentException: Uri lacks 'file' scheme: content://media/picker/0/com.android.providers.media.photopicker/media/1000043213
```

For some reason, after upgrading to Expo 49, seems like the image picker totally changed (the UI) in Android, and it seems like it allows the user to pick photos that the URI are `content://xxxx`

As you can see here, it's only happening since some days ago (when I published SDK 49 upgrade).

<img width="323" alt="Screen Shot 2023-08-29 at 21 59 44" src="https://github.com/expo/expo/assets/1837969/9aef3e41-0f95-4725-b2a6-adea4533eb63">

Is there a workaround for this? Maybe some available flag to switch back to the old UI version, where files with `content://` uris were not visible.


Current code:
```js
    const options = {
      mediaTypes: ImagePicker.MediaTypeOptions.Images,
      allowsEditing: false,
      exif: true,
      quality: 0.91,
    };

    ImagePicker.launchImageLibraryAsync(options)
```

### Environment

```
  expo-env-info 1.0.5 environment info:
    System:
      OS: macOS 12.6.3
      Shell: 5.8.1 - /bin/zsh
    Binaries:
      Node: 16.19.0 - ~/.nvm/versions/node/v16.19.0/bin/node
      Yarn: 1.22.19 - ~/.yarn/bin/yarn
      npm: 8.19.3 - ~/.nvm/versions/node/v16.19.0/bin/npm
    Managers:
      CocoaPods: 1.11.3 - /usr/local/bin/pod
    SDKs:
      iOS SDK:
        Platforms: DriverKit 22.2, iOS 16.2, macOS 13.1, tvOS 16.1, watchOS 9.1
    IDEs:
      Android Studio: 2022.3 AI-223.8836.35.2231.10406996
      Xcode: 14.2/14C18 - /usr/bin/xcodebuild
    npmPackages:
      expo: ^49.0.7 => 49.0.7
      react: 18.2.0 => 18.2.0
      react-native: 0.72.3 => 0.72.3
    npmGlobalPackages:
      eas-cli: 4.1.2
      expo-cli: 6.3.10
    Expo Workflow: managed
```

"expo-image-picker": "~14.3.2",

--- a/packages/expo-image-loader/CHANGELOG.md
+++ b/packages/expo-image-loader/CHANGELOG.md
@@ -10,7 +10,7 @@
 
 ### 🐛 Bug fixes
 
-- Bump `Glide` version to resolve `SecurityException` on Android 13.
+- Bump `Glide` version to resolve `SecurityException` on Android 13. ([#24196](https://github.com/expo/expo/pull/24196) by [@alanjhughes](https://github.com/alanjhughes))
 
 ### 💡 Others
 

--- a/packages/expo-image-loader/CHANGELOG.md
+++ b/packages/expo-image-loader/CHANGELOG.md
@@ -10,12 +10,16 @@
 
 ### 🐛 Bug fixes
 
+- Bump `Glide` version to resolve `SecurityException` on Android 13.
+
 ### 💡 Others
 
 ## 4.3.0 — 2023-06-13
 
 - Removed `com.facebook.fresco:fresco` dependency ([#22542](https://github.com/expo/expo/pull/22542) by [@josephyanks](https://github.com/josephyanks))
+
 ### 📚 3rd party library updates
+
 - Updated `com.github.bumptech.glide:glide` to `4.13.2` and added customizable `glideVersion` variable on ext. ([#22542](https://github.com/expo/expo/pull/22542) by [@josephyanks](https://github.com/josephyanks))
 
 ### 🐛 Bug fixes

--- a/packages/expo-image-loader/android/build.gradle
+++ b/packages/expo-image-loader/android/build.gradle
@@ -86,7 +86,7 @@ repositories {
   mavenCentral()
 }
 
-def glideVersion = safeExtGet('glideVersion', '4.13.2')
+def glideVersion = safeExtGet('glideVersion', '4.16.0')
 
 dependencies {
   implementation project(':expo-modules-core')

